test_scripts/Defects/5_0/885_For_some_RPCs_Core_does_not_unsubscribe_from_softbuttons_after_receiving_a_response/ShowConstantTBT.lua

```diff
@@ -0,0 +1,106 @@
+            }
+        }
+    }
+    tbl.policy_table.functional_groupings["NewTestCaseGroup"] = VDgroup
+    tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.fullAppID].groups = {"Base-4",
+    "NewTestCaseGroup"}
+end
+
+local function ShowConstantTBT()
+    local params = {
+        navigationText1 = "navigationText1",
+        softButtons = {
+            { type = "TEXT", softButtonID = 4, text = "text" }
+        },
+        distanceToManeuver = 50.5,
+        distanceToManeuverScale = 100.5
+    }
+    local responseDelay = 3000
+    local cid = common.getMobileSession():SendRPC("ShowConstantTBT", params )
+    EXPECT_HMICALL("Navigation.ShowConstantTBT",{
+        navigationTexts = {
+            {   fieldName = "navigationText1",
+                fieldText = params.navigationText1,
+            }
+        },
+        distanceToManeuver = 50.5,
+        distanceToManeuverScale = 100.5,
+        appID = common.getHMIAppId(),
+        softButtons = {
+            { type = "TEXT", softButtonID = 4, text = "text" }
+        }
+    })
+    :Do(function(_, data)
+        local function showConstantTBTResponse()
+            common.getHMIConnection():SendResponse(data.id, "Navigation.ShowConstantTBT", "SUCCESS", { })
+        end
+        RUN_AFTER(showConstantTBTResponse, responseDelay)
+    end)
+	:Do(function()
+		local function ButtonEventPress()
+            common.getHMIConnection():SendNotification("Buttons.OnButtonEvent",
+			{ name = "CUSTOM_BUTTON", mode = "BUTTONDOWN", customButtonID = 4, appID = common.getHMIAppId()
+			})
+            common.getHMIConnection():SendNotification("Buttons.OnButtonEvent",
+			{ name = "CUSTOM_BUTTON", mode = "BUTTONUP", customButtonID = 4, appID = common.getHMIAppId()
+			})
+            common.getHMIConnection():SendNotification("Buttons.OnButtonPress",
+			{ name = "CUSTOM_BUTTON", mode = "SHORT", customButtonID = 4, appID = common.getHMIAppId()
+			})
+		end
+
+		RUN_AFTER(ButtonEventPress, 1000)
+	end)
+    common.getMobileSession():ExpectNotification("OnButtonEvent",
+    {buttonName = "CUSTOM_BUTTON", buttonEventMode = "BUTTONDOWN", customButtonID = 4},
+    {buttonName = "CUSTOM_BUTTON", buttonEventMode = "BUTTONUP", customButtonID = 4})
+    :Times(2)
+    common.getMobileSession():ExpectNotification("OnButtonPress",
+    {buttonName = "CUSTOM_BUTTON", buttonPressMode = "SHORT", customButtonID = 4})
+
+    common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("Register App", common.registerApp)
+runner.Step("Update ptu", common.policyTableUpdate, { pTUpdateFunc })
+runner.Step("Activate App", common.activateApp)
+
+-- [[ Test ]]
+runner.Title("Test")
+runner.Step("ShowConstantTBT with soft buttons", ShowConstantTBT)
```

Comment:
@HSavynetska Current test sequence does not check that SDL stops processing softButton events once a response is received for the RPC.
Please correct test sequence.